build.sbt

```diff
@@ -87,6 +87,16 @@ val benchmark =
     )
     .dependsOn(client)
 
+val play =
+  project
+    .in(file("play"))
+    .settings(
+      libraryDependencies ++= Seq(
+        "com.typesafe.play" %% "play" % "2.4.8" % "provided" withSources()
```

Comment:
Also we have to use a different `crossScalaVersions` - as discussed, I'll work on integrating `sbt-doge` for that.